src/Components/Profile.js

```diff
@@ -1,30 +1,27 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-
-
-class Profile extends Component {
-    constructor(props) {
-        super(props);
-
-    }
-
-
-    componentDidMount() {
-
-    }
-
-
-    render() {
-        return (
-            <div>
-
-            </div>
-        );
-    }
-}
-
-Profile.propTypes = {
-
+import React from 'react';
+import { Route } from "react-router-dom";
+import { useState, useEffect } from 'react';
+import ReviewCard from './ReviewCard';
+// import Review from './Review';
+
+const Profile = () => {
```

Comment:
we'll need to pass client as a prop (based on the id of the card that was clicked on the dashboard or search view) to display that client's info